cmd/brimcap/root/command.go

```diff
@@ -53,7 +53,9 @@ type Command struct {
 func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	c := &Command{}
 	c.cli.SetFlags(f)
-	isterm := term.IsTerminal(int(os.Stdout.Fd()))
+	isterm := term.IsTerminal(int(os.Stdout.Fd())) ||
+		term.IsTerminal(int(os.Stdin.Fd())) ||
+		term.IsTerminal(int(os.Stderr.Fd()))
```

Comment:
`-json` only affects stderr, so I wouldn't expect stdin or stdout to influence it.
```suggestion
	isterm := term.IsTerminal(int(os.Stderr.Fd()))
```

Reply:
Makes sense... I think...